packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx

```diff
@@ -0,0 +1,273 @@
+  ref: string;
+  env: string;
+  identifier: string;
+  nextRef: string;
+  accessUrl: string;
+  updatedAt: string;
+  _id: number;
+  isSSR: boolean;
+  healthCheckPath: string;
+  config: {
+    [key: string]: string;
+  };
+  imageUrl: string;
+};
+
+const schema = yup.object().shape({
+  name: yup.string().required(),
+  path: yup.string().required(),
+  imageUrl: yup.string().required(),
+  healthCheckPath: yup.string().required()
+});
+
+type Props = {
+  onClose: () => void;
+  propertyIdentifier: string;
+  dataprops: FormData;
+};
+
+export const ConfigureSSRForm = ({
+  onClose,
+  propertyIdentifier,
+  dataprops
+}: Props): JSX.Element => {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    reset
+  } = useForm<FormData>({
+    resolver: yupResolver(schema)
+  });
+
+  const createSsrSpaProperty = useAddSsrSpaProperty(propertyIdentifier);
+
+  const [keyValuePairs, setKeyValuePairs] = useState([{ key: '', value: '' }]);
+  const handleAddKeyValuePair = () => {
+    setKeyValuePairs([...keyValuePairs, { key: '', value: '' }]);
+  };
+
+  const handleKeyValuePairChange = (index: number, key: string, value: string) => {
+    const newKeyValuePairs = [...keyValuePairs];
+    newKeyValuePairs[index] = { key, value };
+    setKeyValuePairs(newKeyValuePairs);
+  };
+
+  const onSubmit = async (data: FormData) => {
+    const json: Record<string, string> = {};
+    keyValuePairs.forEach(({ key, value }) => {
+      if (key && value) {
+        json[key] = value;
+      }
+    });
+    const temp = data;
+    if (!temp.path.startsWith('/')) {
+      temp.path = `/${temp.path}`;
+    }
+    if (!temp.healthCheckPath.startsWith('/')) {
+      temp.healthCheckPath = `/${temp.healthCheckPath}`;
+    }
+
+    const newData = {
+      ...temp,
+      config: json,
+      propertyIdentifier
+    };
+    try {
+      await createSsrSpaProperty.mutateAsync(newData);
+      onClose();
+      toast.success('Configured and deployed SSR successfully');
+    } catch (error) {
+      toast.error('Failed to deploy SSR');
+    }
+  };
+
+  useEffect(() => {
+    const configKeys = Object.keys(dataprops.config);
+    const configPairs = configKeys.map((key) => ({ key, value: dataprops.config[key] }));
+    setKeyValuePairs(configPairs);
+    reset(dataprops);
+  }, [reset, dataprops]);
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <Split hasGutter>
+        <SplitItem isFilled>
+          <FormGroup label="Name" isRequired>
+            <Controller
+              name="name"
+              control={control}
+              render={({ field }) => (
+                <TextInput
+                  id="name"
+                  type="text"
+                  isDisabled
+                  value={field.value}
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                />
+              )}
+            />
+            {errors.name && <span>{errors.name.message}</span>}
```

Comment:
Pfe already has a convention to show error message. 

Ref: https://www.patternfly.org/v4/components/form/#invalid

Use that and instead of global errors. There is a error state on controller which can be accessed like this

`render{({field,fieldState:{error}})}` use that

Reply:
done